product_abc_classification_sale_stock/tests/test_abc_classification_profile.py

```diff
@@ -255,15 +255,15 @@ def test_01(self):
         # test computed classification and check that inactive products are
         # not taken into account
         self.product1.active = False
-        self.product1.refresh()
+        self.product1.invalidate_model()
```

Comment:
IMO this should be `self.product1.flush_recordset()`

Reply:
![Screenshot from 2024-12-10 12-46-16](https://github.com/user-attachments/assets/09f2ee91-0362-4457-a4dc-501cf6a51217)